### PR TITLE
497 next page

### PR DESCRIPTION
**Pull request by lschatzkin**
I'm reviving this branch as it seemed to have gotten lost in the shuffle. Updating the styles. Still need to have four pages instead of three, and some js to change the colors.

### CODE REVIEW

**Comment by lschatzkin** [comment]
@vbrown608 I've reached a sticking point on this for the "page 4", which is the thank you page.  I'm using jquery to change the progress bar but that page is rendered via an if statement views/tools/_container.html.erb. Is there a way to get that statement to trigger js?

**Comment by esoterik** [comment]
> @vbrown608 I've reached a sticking point on this for the "page 4", which is the thank you page. I'm using jquery to change the progress bar but that page is rendered via an if statement views/tools/_container.html.erb. Is there a way to get that statement to trigger js?

Not sure if this will help, but the thank you page is shown via the various JS files in the `app/assets/javascripts/application/tools` folder -- there's a line to hide the previous page and then show the thank you (`$('#thank-you').show();`). It's code duplication but you could update the bar near there!

**Comment by lschatzkin** [comment]
> > @vbrown608 I've reached a sticking point on this for the "page 4", which is the thank you page. I'm using jquery to change the progress bar but that page is rendered via an if statement views/tools/_container.html.erb. Is there a way to get that statement to trigger js?
> 
> Not sure if this will help, but the thank you page is shown via the various JS files in the `app/assets/javascripts/application/tools` folder -- there's a line to hide the previous page and then show the thank you (`$('#thank-you').show();`). It's code duplication but you could update the bar near there!

Yes, I'm editing the congress_message.js file in that folder... just can't quite get this last bit working.

File: app/assets/javascripts/application/tools/congress_message.js [hunk 1]
```diff
@@ -4,6 +4,43 @@ $(document).on("ready", function() {
...
+    });
+    $(".page-indicator div.page3").css({
+      'background-color': '#BABABA',
+      'color': '#2D2D2D'
+    });
+  });
+
+  $("#tools #congress-message-tool input[type='submit'].test").click(function(){ // show thank you
+
+  });
```
Comment (esoterik) [inline app/assets/javascripts/application/tools/congress_message.js]: oh, sorry, one last thing! this click handler can be deleted now

Comment (lschatzkin) [reply app/assets/javascripts/application/tools/congress_message.js]: Oh, right!

**Pull request merged**